content/documentation/install-uninstall.md

```diff
@@ -26,12 +26,11 @@ The Quire installation process may take a minute or two, during which time there
 
 
 {{< q-class "box warning tip" >}}
```

Comment:
Sorry, just realized, it should be class `box warning` or `box tip`, not both tip and warning at the same time.